api/replication.storage/v1alpha1/volumereplication_types.go

```diff
@@ -36,6 +36,25 @@ const (
 	ConditionValidated = "Validated"
 )
 
+// These are valid messages for various
+// conditions and states of Volume
+// replication.
+const (
+	MessagePromoted               = "volume is promoted"
+	MessageHealthy                = "volume is healthy"
+	MessageNotResyncing           = "volume is not resyncing"
+	MessagePromotionErrorDetected = "error detected while promotion"
```

Comment:
Not clear what does this mean - promotion failed? promotion succeeded with warnings that are safe to ignore?

Same for other "errors detected".

Reply:
it's just a small message indicating that there is some error, and the error will be printed in the reason

Reply:
the promotion failed is mentioned the the condition type promote, I think if we add same message to volume degradation condition it will not make much sense.